test/kubemark/gce/util.sh

```diff
@@ -14,35 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+source test/kubemark/common/util.sh
+
```

Comment:
Another thing is that please use the convention of sourcing files, e.g. like here:
https://github.com/kubernetes/kubernetes/pull/44394/files#diff-a705eb8c1cb0096615ba0ac989e939f1R21